harry-core/src/harry/model/DescriptorSelectorBuilder.java

```diff
@@ -33,7 +33,7 @@ public class DescriptorSelectorBuilder implements Configuration.CDSelectorConfig
 {
     private Function<SchemaSpec, OpSelectors.ColumnSelector> columnSelectorFactory;
     private OpSelectors.OperationSelector operationSelector = DEFAULT_OP_SELECTOR;
-    private Distribution numberOfRowsDistribution = new Distribution.ScaledDistribution(2, 30);
+    private Distribution modificationsPerLts = new Distribution.ScaledDistribution(2, 30);
     private Distribution numberOfModificationsDistribution = new Distribution.ScaledDistribution(1, 3);
     private int maxPartitionSize = Integer.MAX_VALUE;
     private Function<SchemaSpec, int[]> fractionsSupplier = null;
```

Comment:
nit: What does `fractionsSupplier` do?

Reply:
The initial idea was to use HierarchicalDescriptorSelector: sort of to make "predictable" partitions, where parts of CK would repeat.